src/httpclient.c

```diff
@@ -2,23 +2,24 @@
 
 #include <corto/httpclient/httpclient.h>
 #include <curl/curl.h>
-
 #define INITIAL_BODY_BUFFER_SIZE (512)
-
 /* Logging Support */
 corto_buffer g_logBuffer;
```

Comment:
A `corto_buffer` is not a thread-safe API. Calling `corto_buffer` functions from multiple threads will corrupt the buffer. Suggest to add the `g_logBuffer` to the thread specific memory.